src/bioregistry/data/bioregistry.json

```diff
@@ -17815,6 +17815,21 @@
     ],
     "uri_format": "https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=$1"
   },
+  "clingen.curation": {
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "Curations representing evidence aggregation and expert panel assertions based on standardized evaluation procedures",
+    "example": "CCID:004126",
+    "github_request_issue": 1414,
+    "homepage": "https://search.clinicalgenome.org/kb/gene-validity",
+    "name": "ClinGen Curation",
+    "pattern": "^CCID:\\d{6}$",
```

Comment:
CCID should probably be represented as a `banana` and then the identifier pattern/example would not include it